stories/datafusion.stories.js

```diff
@@ -176,3 +176,97 @@ export const getMapProcessingEvalscriptUrl = () => {
 
   return wrapperEl;
 };
+
+export const getMapProcessingGainGamma = () => {
+  if (!process.env.CLIENT_ID || !process.env.CLIENT_SECRET) {
+    return "<div>Please set OAuth Client's id and secret for Processing API (CLIENT_ID, CLIENT_SECRET env vars)</div>";
+  }
+
+  const imgNoGainGamma = document.createElement('img');
+  imgNoGainGamma.width = '256';
+  imgNoGainGamma.height = '256';
+
+  const imgGainIs2 = document.createElement('img');
+  imgGainIs2.width = '256';
+  imgGainIs2.height = '256';
+
+  const imgGammaIs2 = document.createElement('img');
+  imgGammaIs2.width = '256';
+  imgGammaIs2.height = '256';
+
+  const imgGainGammaAre2 = document.createElement('img');
+  imgGainGammaAre2.width = '256';
+  imgGainGammaAre2.height = '256';
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = '<h2>GetMap with Processing and gain and gamma effects</h2>';
+  wrapperEl.innerHTML += '<h4>no gain/gamma | gain | gamma | gain and gamma</h4>';
+  wrapperEl.insertAdjacentElement('beforeend', imgNoGainGamma);
+  wrapperEl.insertAdjacentElement('beforeend', imgGainIs2);
+  wrapperEl.insertAdjacentElement('beforeend', imgGammaIs2);
+  wrapperEl.insertAdjacentElement('beforeend', imgGainGammaAre2);
+
+  const perform = async () => {
+    await setAuthTokenWithOAuthCredentials();
+
+    const layerS2L1C = new S2L1CLayer({ instanceId, layerId: s2l1cLayerId });
+    const layerS2L2A = new S2L2ALayer({
+      instanceId,
+      layerId: s2l2aLayerId,
+      mosaickingOrder: MosaickingOrder.LEAST_RECENT,
+    });
+    const layers = [
+      {
+        layer: layerS2L2A,
+        id: 's2l2a',
+        fromTime: new Date(Date.UTC(2020, 1 - 1, 1, 0, 0, 0)),
+        toTime: new Date(Date.UTC(2020, 2 - 1, 26, 0, 0, 0)),
+      },
+      {
+        layer: layerS2L1C,
+        id: 's2l1c',
+        fromTime: new Date(Date.UTC(2020, 2 - 1, 10, 0, 0, 0)),
+        toTime: new Date(Date.UTC(2020, 2 - 1, 24, 0, 0, 0)),
+      },
+    ];
+    const layer = new ProcessingDataFusionLayer({
+      layers: layers,
+      evalscriptUrl:
+        'https://gist.githubusercontent.com/sinergise-anze/33fe78d9b1fd24d656882d7916a83d4d/raw/295b9d9f033c7e3f1e533363322d84846808564c/data-fusion-evalscript.js',
+    });
+
+    const getMapParams = {
+      bbox: bbox,
+      fromTime: new Date(Date.UTC(2019, 11 - 1, 22, 0, 0, 0)),
+      toTime: new Date(Date.UTC(2020, 2 - 1, 22, 23, 59, 59)),
+      width: 300,
+      height: 300,
+      format: MimeTypes.JPEG,
+    };
+
+    const gain = 2;
+    const gamma = 2;
+    const getMapParamsGainIs2 = { ...getMapParams, gain: gain };
+    const getMapParamsGammaIs2 = { ...getMapParams, gamma: gamma };
+    const getMapParamsGainGammaAre2 = { ...getMapParams, gain: gain, gamma: gamma };
```

Comment:
Maybe it would make sense to use non-deprecated form (`effects`) here?

EDIT: and in S2L2A storybook...

Reply:
Thanks, changed all stories where the gain and gamma are present (BYOC, Data fusion, Envisat Meris, S2L2A, Proba-V)